src/jdk.jfr/share/classes/jdk/jfr/internal/instrument/JDKEvents.java

```diff
@@ -27,36 +27,9 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
 
 import jdk.jfr.Event;
-import jdk.jfr.events.ActiveRecordingEvent;
-import jdk.jfr.events.ActiveSettingEvent;
-import jdk.jfr.events.ContainerIOUsageEvent;
-import jdk.jfr.events.ContainerConfigurationEvent;
-import jdk.jfr.events.ContainerCPUUsageEvent;
-import jdk.jfr.events.ContainerCPUThrottlingEvent;
-import jdk.jfr.events.ContainerMemoryUsageEvent;
-import jdk.jfr.events.DirectBufferStatisticsEvent;
-import jdk.jfr.events.ErrorThrownEvent;
-import jdk.jfr.events.ExceptionStatisticsEvent;
-import jdk.jfr.events.ExceptionThrownEvent;
-import jdk.jfr.events.FileForceEvent;
-import jdk.jfr.events.FileReadEvent;
-import jdk.jfr.events.FileWriteEvent;
-import jdk.jfr.events.DeserializationEvent;
-import jdk.jfr.events.ProcessStartEvent;
-import jdk.jfr.events.SecurityPropertyModificationEvent;
-import jdk.jfr.events.SocketReadEvent;
-import jdk.jfr.events.SocketWriteEvent;
-import jdk.jfr.events.TLSHandshakeEvent;
-import jdk.jfr.events.ThreadSleepEvent;
-import jdk.jfr.events.VirtualThreadStartEvent;
-import jdk.jfr.events.VirtualThreadEndEvent;
-import jdk.jfr.events.VirtualThreadPinnedEvent;
-import jdk.jfr.events.VirtualThreadSubmitFailedEvent;
-import jdk.jfr.events.X509CertificateEvent;
-import jdk.jfr.events.X509ValidationEvent;
+import jdk.jfr.events.*;
```

Comment:
Hello Sean, rest of the changes look fine to me, except this one. Was this an intentional change to use `*` import instead of the explicit ones?

Reply:
This was formatted via IDE. Seems like you've a preference to use the more verbose import style. I'll revert this change.